scripts/check_compliance.py

```diff
@@ -702,15 +737,17 @@ def main():
     for testcase in ComplianceTest.__subclasses__():
         test = testcase(suite, args.commits)
         if args.module:
-            if test._name in args.module:
-                test.run()
-                suite.add_testcase(test.case)
-        else:
-            if test._name in args.exclude_module:
-                print("Skipping {}".format(test._name))
+            if test._name not in args.module:
                 continue
+        elif test._name in args.exclude_module:
+            print("Skipping " + test._name)
+            continue
+
+        try:
             test.run()
-            suite.add_testcase(test.case)
+        except EndTest:
+            pass
+        suite.add_testcase(test.case)
 
     xml = JUnitXml()
     xml.add_testsuite(suite)
```

Comment:
@line 779 would you mind fixing this related and misleading print:

```patch
- print("{} Errors found".format(errors))
+ print("{} checks failed".format(errors))
```
It's really *not* a number of errors, that just really confused me.

Reply:
Fixed